build.sbt

```diff
@@ -1,15 +1,16 @@
-import ProjectSettings.ProjectFrom
+import ProjectSettings._
 import com.typesafe.sbt.packager.docker.Cmd
 
 ThisBuild / scalaVersion      := "2.13.10"
 ThisBuild / organization      := "it.pagopa"
 ThisBuild / organizationName  := "Pagopa S.p.A."
 Global / onChangedBuildSource := ReloadOnSourceChanges
 ThisBuild / dependencyOverrides ++= Dependencies.Jars.overrides
-ThisBuild / version           := ComputeVersion.version
+ThisBuild / version           := "1.0.x-SNAPSHOT"
```

Comment:
is this intended just for testing and be reverted, or should it be reset now to `ComputeVersion.version`?